dipy/reconst/shm.py

```diff
@@ -1060,8 +1060,12 @@ def anisotropic_power(sh_coeffs, norm_factor=0.00001, power=2,
         ap += ap_i
         n_start = n_stop
 
-    # Shift the map to be mostly non-negative:
-    log_ap = np.log(ap) - np.log(norm_factor)
+    # Shift the map to be mostly non-negative,
+    # only applying the log operation to non-zero elements
```

Comment:
I pushed the line-wrapping fix, except if you,re talking about some other line...

Also, maybe I misunderstood... What part of the comment is inaccurate? That we apply only to non_zero voxels? Sorry I'm just not able to pinpoint directly which part is inaccurate.